.github/workflows/release.yml

```diff
@@ -0,0 +1,79 @@
+name: Release
```

Comment:
this is again what was in deploy, but what ran on tag releases - again removing the if conditions now that it only runs when needed